src/openforms/formio/migration_converters.py

```diff
@@ -76,6 +78,68 @@ def set_openforms_datasrc(component: Component) -> bool:
     return True
 
 
+def fix_column_sizes(component: Component) -> bool:
+    component = cast(ColumnsComponent, component)
```

Comment:
can't `isinstance` on a typed dict - normally I'd `# type: ignore` this but there's quite a few more linter warnings :/